src/main/java/mate/academy/spring/mapper/impl/response/OrderResponceMapper.java

```diff
@@ -0,0 +1,19 @@
+package mate.academy.spring.mapper.impl.response;
+
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.Order;
+import mate.academy.spring.model.dto.response.OrderResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class OrderResponceMapper implements DtoResponseMapper<OrderResponseDto, Order> {
+    @Override
+    public OrderResponseDto toDto(Order object) {
```

Comment:
Consider using more descriptive parameter name, like `order` instead of `object`.